app/controllers/api/v1/businesses_controller.rb

```diff
@@ -46,7 +46,7 @@ def update
       # DELETE /businesses/:id
       def destroy
         # soft delete
-        @business.update!(active: false)
+        @business.update!(active: false, deleted_at: Date.today)
```

Comment:
Good call out here in your caveats, @query-string - do you mind changing this behavior to remove setting active to false?

Reply:
Oh also, please use Time.current instead!